datasets/aev-a2d2.yaml

```diff
@@ -0,0 +1,31 @@
+Name: A2D2: AEV Autonomous Driving Dataset
+Description: |
+        An open multi-sensor dataset for autonomous driving research.
+        This dataset comprises semantically segmented images, semantic
+        point clouds, and 3D bounding boxes. In addition, it contains
+        unlabelled 360 degree camera images, lidar, and bus data for
+        three sequences. We hope this dataset will further facilitate
+        active research and development in AI, computer vision, and
+        robotics for autonomous driving. Commercial use is permitted
+        as per the CC BY-ND 4.0 license.
+Contact: aevdrivingdataset@audi.de
+Documentation: https://www.audi-electronics-venture.de/driving-dataset.html
+ManagedBy: Audi Electronics Ventures GmbH, 85080 Gaimersheim, Germany
+UpdateFrequency: |
+        The dataset may be updated with additional or corrected data
+        on a need-to-update basis.
+Tags: |
```

Comment:
This should be like

Tags:
  - tag 1
  - tag 2

Please make sure to add `aws-pds` as a tag. Also, please use tags from https://github.com/awslabs/open-data-registry/blob/master/tags.yaml and if we are missing any major tags there, please let us know.

Reply:
Could we add 'slam' and 'perception' tags to this list?

Reply:
I think since these tags would currently only apply to this dataset, we do not want to add them. We can reevaluate if/when we get more similar datasets? From a practical standpoint, if you include these words in the description, they will get picked up by the search the same as the tags. Does that sound good?

Reply:
That is fine by us. Thank you.